### PR TITLE
Module 4

### PR DESCRIPTION
**Pull request by SChandarana**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: ansible/vm_files/playbook.yml
Comment (SChandarana) [inline ansible/vm_files/playbook.yml:55]: This won't work until this branch is merged as the todoapp.service hasn't been pushed to main (I was using this git branch when testing it all worked but the final playbook uses the main branch when it clones git)

**Review by SChandarana — COMMENTED** [review]
Just a quick question, in previous tasks I removed the flask config file (as we were no longer using sessions) Is this fine or will I need to bring it back at some point :D

File: ansible/vm_files/playbook.yml
Comment (JackMeadDev) [inline ansible/vm_files/playbook.yml:31]: Not a big issue, but worth trying to avoid getting in the habit of `777` for everything! `755` is quite common for directories, and will just limit write access

File: ansible/vm_files/playbook.yml
Comment (JackMeadDev) [reply ansible/vm_files/playbook.yml:55]: Makes sense!

File: README.md
Comment (JackMeadDev) [inline README.md]: Minor, but it would be nice to also mention how to run the ansible commands on the control node here

**Review by JackMeadDev — APPROVED** [review]
This looks great, nice & clean solution. 😄 I've left a couple of very minor optional suggestions below, but happy for you to merge it when you're ready (and I think we've already cleaned up your VMs, so you won't be able to test any changes!)

In answer to your separate question - removing the `flask_config` file is a good thing; we will reintroduce the `SECRET_KEY` variable at a later date but you can bring it back then or just set the value directly on the app so not a problem.

**Pull request merged**
